cmd/neofs-node/netmap.go

```diff
@@ -165,6 +169,29 @@ func initNetmapService(c *cfg) {
 	}
 }
 
+func readSubnetCfg(c *cfg) {
+	var subnetCfg nodeconfig.SubnetConfig
+
+	subnetCfg.Init(*c.appCfg)
+
+	var (
+		id  subnetid.ID
+		err error
+	)
+
+	subnetCfg.IterateSubnets(func(idTxt string) {
+		err = id.UnmarshalText([]byte(idTxt))
+		fatalOnErrDetails("parse subnet entry", err)
+
+		c.cfgNodeInfo.localInfo.EnterSubnet(id)
```

Comment:
what if `exit_zero: true` and `entries` contains `0`?

Reply:
`exit_zero` overrides attributes and `entries` (see comment in example config).

Reply:
i just meant that it looks confusing. error instead of overriding